test/postman.test.ts

```diff
@@ -166,4 +166,35 @@ describe("Postman", function () {
 
         expect(userL2Balance).to.deep.equal({ balance: 92n });
     });
+
+    it("Should mock l1 to l2 mock tx and vice versa", async () => {
+        const L1_CONTRACT_ADDRESS = l1l2Example.address;
+
+        const l1ToL2Response = await starknet.devnet.sendMessageToL2(
+            l2contract.address,
+            "deposit",
+            L1_CONTRACT_ADDRESS,
+            [1, 1],
+            0
+        );
+
+        expect(l1ToL2Response).haveOwnProperty("transaction_hash");
+        await account.invoke(l2contract, "increase_balance", {
+            user,
+            amount: 10000000
+        });
+
+        await account.invoke(l2contract, "withdraw", {
```

Comment:
What is the purpose of these two invokes?

Reply:
These invokes are analogical to our test in devnet -> `def test_consume_message_from_l2_deploy_execute()`